drv/lpc55-update-server/src/main.rs

```diff
@@ -1027,6 +1100,71 @@ impl ServerImpl<'_> {
 
         Ok(())
     }
+
+    /// Cancel pending transient or persistent Hubris image selection.
+    fn switch_default_hubris_image_cancel_pending(
```

Comment:
This name confused me because it sounds like it's switching the default image, not clearing the pending. Maybe `fn clear_pending_image_switch`?